### PR TITLE
Android.mk: add GME support

### PR DESCRIPTION
**Pull request by mechakotik**

### CODE REVIEW

File: Android.mk [hunk 1]
```diff
@@ -136,6 +145,12 @@ ifeq ($(SUPPORT_WAVPACK),true)
     LOCAL_STATIC_LIBRARIES += wavpack
 endif
 
+ifeq ($(SUPPORT_GME),true)
+    LOCAL_CFLAGS += -DMUSIC_GME
+    LOCAL_C_INCLUDES += $(LOCAL_PATH)/$(GME_LIBRARY_PATH)
```
Comment (sezero) [inline Android.mk:150]: Do you not need to append `/gme` here ?

Comment (mechakotik) [reply Android.mk:150]: SDL_mixer source includes it like `#include <gme/gme.h>`, so it's correct include path.

Comment (sezero) [reply Android.mk:150]: Ah yes. Sorry for the noise.

**Comment by sezero** [comment]
Patch is in, and cherry-picked to SDL2 branch too. Thanks.

**Pull request merged**